lupus_to_mqtt/sensor/__init__.py

```diff
@@ -31,6 +31,8 @@ def __init__(self, data, panel):
         self._is_doorbell = CONST.RESPONSE_DOORBELL in self._resp_mode
 
         self._panel = panel
+        self._level = 0
+        self._onOff = 0
```

Comment:
Could you move the level from the general class over to the "Shutter" class as it's only used there?